README.md

```diff
@@ -155,6 +149,18 @@ This knife plugins requires >= Chef 11.0.0. More details about Knife plugins can
 
 ## Nodes
 
+### Bootstrapping Windows Nano Server
+
+Windows Nano Server cannot install Chef via an MSI. Instead Nano supports the `appx` package format. Until Omnitruck strarts serving appx packages automatically to nano, you will need to provide an appx package URL to the `--msi-url` argument:
```

Comment:
it's not clear to me as a non-chef employee if I have to create these packages or ask someone to build them for me